src/main/java/org/folio/rest/impl/HoldingsStorageAPI.java

```diff
@@ -307,7 +308,7 @@ public void putHoldingsStorageHoldingsByHoldingsRecordId(
                   if (Objects.equals(entity.getHrid(), existingHoldings.getHrid())) {
                     try {
                       postgresClient.startTx(connection -> {
-                        updateItemEffectiveCallNumbersByHoldings(connection, postgresClient, entity).onComplete(updateResult -> {
+                        updateItemEffectiveCallNumbersAndLocationsByHoldings(connection, postgresClient, entity).onComplete(updateResult -> {
```

Comment:
I'd change it to updateRelatedItemsAttributes for brevity.

Reply:
I might propose `updateItemEffectiveProperties` or `deriveItemEffectiveProperties` instead.